src/mupdfjs.ts

```diff
@@ -51,8 +51,15 @@ export class StrokeState extends mupdf.StrokeState { }
 export class StructuredText extends mupdf.StructuredText { }
 export class Text extends mupdf.Text { }
 
+export type PDFWord = {
```

Comment:
Please move this type definition down below the definition for `CreatableAnnotationType` ( just want to store all the type definitions in the one place )